edgelight/edgelight.ino

```diff
@@ -38,218 +36,226 @@ SOFTWARE.
 #include "Base64.h"
 
 #ifdef ESP32
-  #include <SPIFFS.h>
+#include <SPIFFS.h>
 #endif
 
 #include <ArduinoJson.h> // https://github.com/bblanchon/ArduinoJson
 
-// TODO : both of these should be in the config file
-#define DELAY_BETWEEN_QUERIES 30000 // 30 seconds
-#define TARGET_CONFIDENCE 0.9 // 90% confidence for queries
+#include "config.h"
 
 #define FORMAT_SPIFFS_IF_FAILED true
 
 // setting this will allow a GPIO held on boot to reset the wifi credentials and file system
 #define GPIO_RESET_CREDENTIALS -1
 
 // set the board type here.  check the pin definitions below to match your actual board
+// #define CAMERA_MODEL_ESP32_CAM_BOARD 1
 #define CAMERA_MODEL_M5STACK_PSRAM 1
 
 #if defined(CAMERA_MODEL_ESP32_CAM_BOARD)
-#define PWDN_GPIO_NUM     32
-#define RESET_GPIO_NUM    -1
-#define XCLK_GPIO_NUM      0
-#define SIOD_GPIO_NUM     26
-#define SIOC_GPIO_NUM     27
-
-#define Y9_GPIO_NUM       35
-#define Y8_GPIO_NUM       34
-#define Y7_GPIO_NUM       39
-#define Y6_GPIO_NUM       36
-#define Y5_GPIO_NUM       21
-#define Y4_GPIO_NUM       19
-#define Y3_GPIO_NUM       18
-#define Y2_GPIO_NUM        5
-#define VSYNC_GPIO_NUM    25
-#define HREF_GPIO_NUM     23
-#define PCLK_GPIO_NUM     22
-
-#define GPIO_LED_FLASH    4
+#define PWDN_GPIO_NUM 32
```

Comment:
the autoformatter has poor taste in typesetting !

Reply:
yeah it just bulldozed all that nice alignment!